UITests/TopTabsTests.swift

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+@testable import Storage
+@testable import Client
+
+class TopTabsTests: KIFTestCase {
+    private var webRoot: String!
+    let numberOfTabs = 4
+    
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+    
+    func testTopTabs() {
```

Comment:
Xcode will automatically recognize any functions that start with test* as test cases so you don't need to run them all from a since function.